src/SFA.DAS.ApplyService.InternalApi.UnitTests/ExperienceAndAccreditationControllerTests.cs

```diff
@@ -196,75 +199,93 @@ public void get_gateway_declaration_contract_file_clarification_returns_the_subm
+                GetConstructedPage(
+                    RoatpWorkflowPageIds.ExperienceAndAccreditations.GradeWithinLast3YearsOfsFunded,
+                    RoatpYourOrganisationQuestionIdConstants.GradeWithinLast3YearsOfsFunded,
+                    "Yes"),
+                GetConstructedPage(
+                    RoatpWorkflowPageIds.ExperienceAndAccreditations.HasMonitoringVisitGradedInadequateInLast18Months,
+                    RoatpYourOrganisationQuestionIdConstants.HasMonitoringVisitGradedInadequateInLast18Months,
+                    "Yes"),
+            };
 
-            _qnaApiClient
-                .Setup(x => x.GetAnswerValueFromActiveQuestion(_applicationId,
-                    RoatpWorkflowSequenceIds.YourOrganisation,
-                    RoatpWorkflowSectionIds.YourOrganisation.ExperienceAndAccreditations,
-                    It.Is<PageAndQuestion[]>(y =>
-                        y.First().PageId == RoatpWorkflowPageIds.ExperienceAndAccreditations.GradeWithinLast3YearsOfsFunded && y.First().QuestionId == RoatpYourOrganisationQuestionIdConstants.GradeWithinLast3YearsOfsFunded
-                        && y.Last().PageId == RoatpWorkflowPageIds.ExperienceAndAccreditations.GradeWithinLast3YearsNonOfsFunded && y.Last().QuestionId == RoatpYourOrganisationQuestionIdConstants.GradeWithinLast3YearsNonOfsFunded)))
-                .ReturnsAsync("Yes");
 
+           
+            
+            
+            
+            
+            
+
+            applicationSection.QnAData.Pages = pages;
+            ;
+            
+           
+            // _qnaApiClient
+            //     .Setup(x => x.GetAnswerValueFromActiveQuestion(_applicationId,
+            //         RoatpWorkflowSequenceIds.YourOrganisation,
+            //         RoatpWorkflowSectionIds.YourOrganisation.ExperienceAndAccreditations,
+            //         RoatpWorkflowPageIds.ExperienceAndAccreditations.HasMonitoringVisitGradedInadequateInLast18Months,
+            //         RoatpYourOrganisationQuestionIdConstants.HasMonitoringVisitGradedInadequateInLast18Months)).ReturnsAsync("Yes");
+            //
+            //
+            // _qnaApiClient
+            //     .Setup(x => x.GetAnswerValueFromActiveQuestion(_applicationId,
+            //         RoatpWorkflowSequenceIds.YourOrganisation,
+            //         RoatpWorkflowSectionIds.YourOrganisation.ExperienceAndAccreditations,
+            //         It.Is<PageAndQuestion[]>(y =>
+            //             y.First().PageId == RoatpWorkflowPageIds.ExperienceAndAccreditations.GradeWithinLast3YearsOfsFunded && y.First().QuestionId == RoatpYourOrganisationQuestionIdConstants.GradeWithinLast3YearsOfsFunded
+            //             && y.Last().PageId == RoatpWorkflowPageIds.ExperienceAndAccreditations.GradeWithinLast3YearsNonOfsFunded && y.Last().QuestionId == RoatpYourOrganisationQuestionIdConstants.GradeWithinLast3YearsNonOfsFunded)))
+            //     .ReturnsAsync("Yes");
```

Comment:
big block of commented out code here. Can we delete it?

Reply:
sorrywill do